### ISSUE TITLE
Attempted import error: 'IdProvider' is not exported from '@radix-ui/react-id' (imported as 'Ne').

### ISSUE THREAD

**Issue by armenic** [aws-amplify/amplify-ui#967]
### Before creating a new issue, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-ui/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-ui/discussions).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md#bug-reports).

### On which framework/platform are you having an issue?

React

### Which UI component?

Other

### Please describe your bug.

importing anything from `@aws-amplify/ui-react` causes the following error. For example in the `App.js` I have:

```
import { Button } from "@aws-amplify/ui-react";
```

which causes the error

```
./node_modules/@aws-amplify/ui-react/dist/esm/index.js
Attempted import error: 'IdProvider' is not exported from '@radix-ui/react-id' (imported as 'Ne').
```

### What's the expected behaviour?

I expect no error

### Help us reproduce the bug!

Basically the steps in https://docs.amplify.aws/start/getting-started/installation/q/integration/react/
```
npm install -g @aws-amplify/cli
amplify configure
npx create-react-app react-amplified
amplify init
npm install aws-amplify @aws-amplify/ui-react
```

### Code Snippet

```typescript
import logo from './logo.svg';
import './App.css';

import { Button } from "@aws-amplify/ui-react";

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <p>
          Edit <code>src/App.js</code> and save to reload.
        </p>
        <a
          className="App-link"
          href="https://reactjs.org"
          target="_blank"
          rel="noopener noreferrer"
        >
          Learn React
        </a>
      </header>
    </div>
  );
}

export default App;


```

### Additional information and screenshots

package.json
```json
{
  "name": "react-amplified",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "@aws-amplify/ui-react": "^2.1.3",
    "@testing-library/jest-dom": "^5.16.1",
    "@testing-library/react": "^11.2.7",
    "@testing-library/user-event": "^12.8.3",
    "aws-amplify": "^4.3.10",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-scripts": "4.0.3",
    "web-vitals": "^1.1.2"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
  "eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest"
    ]
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
}

```

**Comment by alharris-at** [comment]
Related to the issue here https://github.com/radix-ui/primitives/issues/1026. We are working on a fix that we hope to release shortly.

As a workaround @armenic, we've verified that manually adding `"@radix-ui/react-id": "0.1.1"` to your `package.json` file,  deleting your package-lock and node_modules and rerunning `yarn` or `npm install` seems to address this issue. Could you confirm?

**Comment by hvergara** [comment]
Hey @armenic! we just published a new release (2.1.4), which pins the `@radix-ui/*` packages to `0.1.1`. Can you confirm if installing a new version of `@aws-amplify/ui-react` solves the issue on your end?

`npm install @aws-amplify/ui-react`

**Comment by armenic** [comment]
@hvergara, you guys are awesome, thanks you so much for fixing it so super quickly!
